src/hooks/wallets/useOnboard.ts

```diff
@@ -137,14 +139,15 @@ export const useInitOnboard = () => {
   const { configs } = useChains()
   const chain = useCurrentChain()
   const onboard = useStore()
+  const customRpc = useAppSelector(selectRpc)
 
   useInitPairing()
 
   useEffect(() => {
     if (configs.length > 0) {
-      initOnboard(configs)
+      void initOnboard(configs, customRpc)
```

Comment:
why adding `void` ?

Reply:
`initOnboard` returns a promise that we don't intend to use. Thats why we don't assign it to anything. This is not good practice and could have unintended side-effects as described here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/void#non-leaking_arrow_functions

(also, my IDE is complaining)

Reply:
Gotcha! Thank you